src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -375,6 +376,8 @@ public ListBoxModel doFillGitToolItems() {
             return gitScmDescriptor.doFillGitToolItems();
         }
 
+        @SuppressFBWarnings(value = "NP_PARAMETER_MUST_BE_NONNULL_BUT_MARKED_AS_NULLABLE",
+                            justification = "stripToEmpty(projectName).length() <= 2 will ensure that projectName is not null")
         @POST
         public HttpResponse doFillProjectNameItems(@Nullable @QueryParameter String serverId,
```

Comment:
putting on a philosophical hat here. We say "this value can be null, and we enforce the non-nullness of it by extra checks". Isn't the annotation actually wrong? It seems to me that this should actually be a Nonnull (which given package-level annotation means there should be no annotations here).